lib/bitex_bot/models/api_wrappers/itbit/itbit_api_wrapper.rb

```diff
@@ -9,16 +13,9 @@ def self.setup(settings)
     end
   end
 
-  def self.transactions
-    Itbit::XBTUSDMarketData.trades.map { |t| transaction_parser(t.symbolize_keys) }
-  end
-
-  def self.orders
-    Itbit::Order.all(status: :open).map { |o| order_parser(o) }
-  end
-
-  def self.order_book
-    order_book_parser(Itbit::XBTUSDMarketData.orders)
+  def self.amount_and_quantity(order_id)
```

Comment:
This is broken. `ClosingFlow#sync_positions` will pass two arguments: `order_id` and `transactions`.

Reply:
fixed.